polkadot-parachains/res/rockmine.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Westmint",
+  "id": "westmint",
```

Comment:
that seems wrong :-D
also agree on only adding this once we have a spec that we're running
as we'll upgrade shell to Rockmine we'll start with a shell Chainspec